### PR TITLE
doc: document for statement `optimize table`

### PR DESCRIPTION
**Pull request by dantengsky**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Documentations for statement `optimize table`

## Changelog

- Documentation
- Not for changelog (changelog entry is not required)

## Related Issues

Fixes #issue

### CODE REVIEW

**Comment by dantengsky** [comment]
@soyeric128 

A draft doc for the statement `optimize table` is provided, please have a look & thanks!

File: docs/doc/30-reference/30-sql/00-ddl/20-table/60-optimize-table.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+   The same as `optimize table T purge`
+
+
+
+
+## Examples
+
+```sql
+
```
Comment (Xuanwo) [inline docs/doc/30-reference/30-sql/00-ddl/20-table/60-optimize-table.md:43]:
```suggestion
```

File: docs/doc/30-reference/30-sql/00-ddl/20-table/60-optimize-table.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+title: OPTIMIZE TABLE
+---
+
+Optimize the give table, by compacting data, purging historical data.
```
Comment (dantengsky) [inline docs/doc/30-reference/30-sql/00-ddl/20-table/60-optimize-table.md:5]:
```suggestion
Optimize the given table, by compacting data, and purging historical data.
```

File: docs/doc/30-reference/30-sql/00-ddl/20-table/60-optimize-table.md (hunk 1 above)
Comment (dantengsky) [reply docs/doc/30-reference/30-sql/00-ddl/20-table/60-optimize-table.md:43]: thanks!

**Pull request merged**